src/models/Projection.ts

```diff
@@ -49,98 +28,47 @@ export class Projection {
   daysSinceDayZero: number;
   hospitalizations: number[];
   beds: number[];
-  deaths: number[];
```

Comment:
deaths and infected are only available as cumulative in the API.  We could calculate the deltas if we need them, but right now we don't.